src/Map/src/Renderer/AbstractRenderer.php

```diff
@@ -39,6 +51,8 @@ final public function renderMap(Map $map, array $attributes = []): string
             $map->options($defaultMapOptions);
         }
 
+        $map->options($this->tapOptions($map->getOptions()));
```

Comment:
Not sure to get the name

Reply:
Inspired from https://laravel.com/docs/11.x/helpers#method-tap, this idea is to:
- accept a parameter
- do something to this parameter
- return the parameter